### PR TITLE
fix(jobCard): Use pseudo element to reduce browser reprint

### PR DESCRIPTION
**Pull request by patrickliu-sa**
Refactored `sideHighlightBorder` with using `::after` and opacity change to prevent reprint.

Since the pseudo element is on the padding area, it might help to revoke the complex calculation of inset padding for `leftContainer` and `rightContainer` (or even revoke those changes made previously)

It's an extended UI update from #393

### CODE REVIEW

**Comment by patrickliu-sa** [comment]
@youwei91 could you take a look on this one as it's blocking #385 created by @kosanna ? 
🙇

File: react/JobCard/JobCard.less [hunk 1]
```diff
@@ -48,24 +48,39 @@
...
+    background-color: transparent;
+    bottom: 0;
+    content: '';
+    display: block;
+    left: 0;
+    opacity: 0;
+    position: absolute;
+    top: 0;
+    width: @grid-base;
+  }
```
Comment (youwei91) [inline react/JobCard/JobCard.less]: above changes is refer to the requirement?

File: react/JobCard/JobCard.less [hunk 2]
```diff
@@ -170,11 +186,17 @@
 }
 
 .sideHighlightBorder-jobsdb {
```
Comment (youwei91) [inline react/JobCard/JobCard.less]: Below changes is referring any requirement?

File: react/JobCard/JobCard.less (hunk 1 above)
Comment (patrickliu-sa) [reply react/JobCard/JobCard.less]: Use opacity change for `sideHighLightBorder` color when needed can greatly reduce trigger of layout repaint, which is very expansive especially on SRP with multiple card components.

Ref: https://developers.google.com/web/fundamentals/design-and-ux/animations/animations-and-performance#tldr

File: react/JobCard/JobCard.less (hunk 2 above)
Comment (patrickliu-sa) [reply react/JobCard/JobCard.less]: I tried to retain the original padding even when branded ad style triggered

**Comment by mengtzu** [comment]
Looks like the spacing is different after this change (the text shifts closer to the left hand edge) - is that intended?

**Comment by patrickliu-sa** [comment]
> Looks like the spacing is different after this change (the text shifts closer to the left hand edge) - is that intended?

It should fixed now.
Preview: https://seek-asia-style-guide--daa2fe27f58c887610b4518cfd726ea3d1a7dbe2.surge.sh/jobCard

**Comment by kosanna** [comment]
Check with @youwei91 if you're not clashing with his PR

**Comment by patrickliu-sa** [comment]
Closing this PR as there are ongoing jobCard changes.  This PR might becomes a potential blocker or messing up with those updates.

**Pull request closed without merging**